### ISSUE TITLE
Очередность платежа  в платежном поручении

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#1882]
Originally reported on Google Code with ID 1882

```
С 1 января 2010 года изменились условные обозначения очередностей платежей.
Сейчас при добавлении новой платежки надо по-умолчанию подставлять значение
<30> (а не <42>).
http://infobank.by/328/itemid/4447/Default.aspx
```

Reported by `danilchyk` on 2010-02-23 11:56:39

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2010-02-23 12:28:52
- **Labels added**: Bank

**Comment by gsbelarus** [comment]
Reported by `alexandra.gsoftware` on 2011-02-25 18:37:53
- **Status changed**: `Duplicate`
- **Merged into**: #2060
